metrics/metrics-core/src/main/java/io/servicecomb/metrics/core/monitor/InvocationMonitor.java

```diff
@@ -20,27 +20,49 @@
 import com.netflix.servo.monitor.BasicCounter;
 import com.netflix.servo.monitor.MonitorConfig;
 
+import io.servicecomb.metrics.core.MetricsConst;
+import io.servicecomb.metrics.core.metric.ConsumerInvocationMetric;
 import io.servicecomb.metrics.core.metric.InvocationMetric;
+import io.servicecomb.metrics.core.metric.ProducerInvocationMetric;
+import io.servicecomb.swagger.invocation.InvocationType;
 
 public class InvocationMonitor {
   private final String operationName;
 
+  private final String consumerPrefix;
+
+  private final String producerPrefix;
+
   private final BasicCounter waitInQueue;
 
   private final TimerMonitor lifeTimeInQueue;
 
   private final TimerMonitor executionTime;
 
+  private final TimerMonitor producerLatency;
+
   private final TimerMonitor consumerLatency;
 
-  private final TimerMonitor producerLatency;
+  private final CallMonitor producerCall;
```

Comment:
We still have the prducerXXX and consumerXXX here!

Reply:
Because can't known invocation type before starting process ,all monitor temp keep in this class; when it convert to metric while polling ,will be separate into ConsumerMetric and ProducerMetric.
After Java Chassis improved invocation generation logic, will be refactor this code immediately